docs/cel2/operators/run-node.md

```diff
@@ -317,7 +351,11 @@ Please reach out to our team on [Discord](https://chat.celo.org) in the [#celo-L
 
 ### Transactions are not being executed when submitted to a node
 
-If your node is synced but transtransactions submitted to it are not executed, make sure the `--rollup.sequencerhttp=https://sequencer.alfajores.celo-testnet.org` flag is correctly set.
+If your node is synced but transactions submitted to it are not executed, make sure the `--rollup.sequencerhttp` flag is correctly set.
+
+- Mainnet: `--rollup.sequencerhttp=https://sequencer.cel2-sequencer-mainnet.celo-testnet.org`
```

Comment:
This URL is not for the real mainnet sequencer, is it?

Reply:
Removed it for now